src/main.cpp

```diff
@@ -638,22 +638,110 @@ unsigned int LimitOrphanTxSize(unsigned int nMaxOrphans) EXCLUSIVE_LOCKS_REQUIRE
     return nEvicted;
 }
 
-bool IsFinalTx(const CTransaction &tx, int nBlockHeight, int64_t nBlockTime)
+int64_t LockTime(const CTransaction &tx, int flags, const CCoinsView* pCoinsView, int nBlockHeight, int64_t nBlockTime)
```

Comment:
Luke, did you notice that the semantics changed? That's why the name
changed.
On Aug 18, 2015 7:23 PM, "Luke-Jr" <notifications@github.com> wrote:

> In src/main.cpp
> <https://github.com/bitcoin/bitcoin/pull/6312#discussion_r37374581>:
>
> > @@ -638,22 +638,110 @@ unsigned int LimitOrphanTxSize(unsigned int nMaxOrphans) EXCLUSIVE_LOCKS_REQUIRE
> >      return nEvicted;
> >  }
> >
> > -bool IsFinalTx(const CTransaction &tx, int nBlockHeight, int64_t nBlockTime)
> > +int64_t LockTime(const CTransaction &tx, int flags, const CCoinsView* pCoinsView, int nBlockHeight, int64_t nBlockTime)
>
> IsFinalTx was a better name IMO.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bitcoin/bitcoin/pull/6312/files#r37374581>.
>